### PR TITLE
feat(jwt) log jwt validation errors

### PR DESCRIPTION
**Pull request by horymury**
Logs the jwt errors both from xmpp and from a client-side validation.

### CODE REVIEW

File: react/features/base/jwt/constants.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+export const MEET_FEATURES = [
```
Comment (saghul) [inline react/features/base/jwt/constants.js:1]: Document?

Comment (saghul) [reply react/features/base/jwt/constants.js:1]: Also, please sort these alphabetically.

File: connection.js [hunk 2]
```diff
@@ -179,6 +182,21 @@ function connect(id, password, roomName) {
             APP.store.dispatch(setPrejoinDisplayNameRequired());
         }
 
+        /**
+         * Logs jwt validation errors from xmpp and from the client-side validator.
+         *
+         * @param {string} message -The error message from xmpp.
+         * @returns {void}
+         */
+        function _logJwtErrors(message) {
```
Comment (saghul) [inline connection.js:191]: Some of the validations here apply only to JaaS users, please let's make that clear so other Jitsi users whi authenticate with a JWT don't see unexpected errors.

File: connection.js [hunk 3]
```diff
@@ -124,7 +125,9 @@ function connect(id, password, roomName) {
          *
          */
         function connectionFailedHandler(error, message, credentials, details) {
-        /* eslint-enable max-params */
+            _logJwtErrors(message);
```
Comment (saghul) [inline connection.js:128]: What about mobile?

**Review by saghul — CHANGES_REQUESTED** [review]

File: connection.js (hunk 2 above)
Comment (horymury) [reply connection.js]: aud, iss and sub ~ kid validations are now JaaS only.

File: connection.js (hunk 3 above)
Comment (horymury) [reply connection.js]: Moved the logging under the `base/conferencemiddleware.any.js`  handler for connection failed.

**Review by saghul — APPROVED** [review]

**Pull request merged**